index.html

```diff
@@ -1,480 +1,225 @@
 <!DOCTYPE html>
-
-<html>
-
-  <head>
-
-    <meta charset="UTF-8">
-
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <title>Johnson's Portfolio</title>
-
-    <link rel="stylesheet" href="styles.css">
-
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-
-    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Montserrat:wght@400;700&family=Playfair+Display:wght@400;500&family=Poppins:wght@100;300;400&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
-  
-  </head>
-
-  <body>
-
-    <header>
-      <nav>
-        <ul>
-          <li>
-            <a href="#homepage">Home</a>
-          </li>
-          <li>
-            <a href="#">Portfolio</a>
-          </li>
-          <li>
-            <a href="#">About</a>
-          </li>
-          <li>
-            <a href="#">Contact</a>
-          </li>
-        </ul>
-        <a class="menu-burger" href="#"><img src="icons/ic_menu.png"></a>
-      </nav>
-    </header>
-
-
-    <main>
-      <!-- start home page section  -->
-      <section class="homepage">
-
-        <div class="container">
-
-          <h1>
-
-            Johnson Macharia
-
-          </h1>
-
-          <p>
-            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br><br>
-            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Johnson Macharia</title>
+  <link href="./styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;600;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="#homepage">Home</a></li>
+        <li><a href="#projects-container">Portfolio</a></li>
+        <li><a href="#about-me-container">About</a></li>
+        <li><a href="#contact-form">Contact</a></li>
+      </ul>
+      <a class="menu" href="#"><img src="icons/ic_menu.png"></a>
+    </nav>
+
+    <div id="mobile_menu">
```

Comment:
- [ ] 👏 Good job on creating the mobile menu! However, I noticed that the current styling for the mobile menu has `width: 100vw` and `height: 117vh`, which is causing a horizontal scroll due to the extra height. To avoid the scroll, I recommend updating the height to `100vh`.

[Johnson-Macharia (2).webm](https://github.com/JohnteFresh/Portfolio-Development/assets/55840999/e7f9a708-c0f0-481d-9c52-6bead18f4a6d)

Please make this adjustment in your code to ensure that the mobile menu fits perfectly within the viewport without any unwanted scrolling. Keep up the great work! 🌟💻🚀